README.md

```diff
@@ -129,6 +129,33 @@ env "gorm" {
 }
 ```
 
+### Supported Features
+#### View
+To create a view, you can define the `ViewDef` method on a struct:
+
+```go
+type User struct {
+  gorm.Model
+  Name string
+  Age  int
+}
+
+type WorkingAgedUsers struct {}
+
+func (WorkingAgedUsers) ViewDef(db *gorm.DB) gorm.ViewOption {
+	return gorm.ViewOption{
+		Query: db.Table("users").Select("name", "age").Where("age > ?", 18),
+	}
+}
+```
+Then using `WithViews` option to load the view:
+
+```go
+stmts, err := gormschema.New("mysql",
+  gormschema.WithViews(&models.WorkingAgedUsers{}),
+).Load(&models.User{})
```

Comment:
API looks weird. By default, the loader should auti-detect models that are marked as Views and generate the SQL for them. The Go API of this should also be consistent - we load all models/views/triggers at once. It does not make sense passing views as loader options.